apps/docs/stories/components/CheckoutWrapper/Docs.mdx

```diff
@@ -43,6 +42,71 @@ Optional:
 - [justifi-billing-form](/docs/modular-checkout-billing-form--docs)
 - [justifi-postal-code-form](/docs/modular-checkout-postal-code-form--docs)
 
+> **Note:** In addition to the supported sub components, you can also include any custom HTML elements or markup inside the `checkout-wrapper`. This allows you to add headings, descriptions, disclaimers, or other layout elements as needed.
+
+## Providing Billing Information
+
+**For `justifi-card-form`:**
+
+Billing information **is required** for checkout completion and can be provided in one of the following ways:
+
+- Using the `<justifi-billing-form />` component
+
+- Using the `<justifi-postal-form />` component
+
+- Passing a `billingInformation` object directly to the `submitCheckout` method
+
+The billingInformation object can take one of two forms:
+
+**Full billing address:**
+
+```javascript
+{
+  name: string;
+  address_line1: string;
+  address_line2?: string;
+  address_city: string;
+  address_state: string;
+  address_postal_code: string;
+}
+```
+
+**Only postal code:**
+
+```javascript
+{
+  address_postal_code: string;
+}
+```
+
+**For `<justifi-bank-account-form />`:**
+
+Billing information **must be provided** through either:
+
+- The `<justifi-billing-information-form />` component
+
+- The full `billingInformation` object passed directly
+
+## Completing Checkout
+
+To finalize the checkout process, include a submit button that calls the `submitCheckout` method on the wrapper. This method.
+
+- Handles validation, form submission and payment method tokenization.
+
+- Optionally accepts a `billingInformation` object (useful when not using a billing component)
+
+- Emits a `checkout-complete-event` when the submission is successful
+
+- This method **does not** return a promise
+
+> If you are using a new payment method and want to save it for future use, set the `savePaymentMethod` prop to `true` on the wrapper. This will automatically save the payment method after successful checkout.
```

Comment:
I'd change this to: 
If you passed a `payment_method_group_id` to the [create checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts/operation/CreateCheckout) and you want to give the user the option to save a newly entered payment method to that payment method group for future use, set the `savePaymentMethod` prop to `true` on the `justifi-checkout-wrapper`. This will automatically save the payment method to that payment method group after successful checkout.